05-onchain.md

```diff
@@ -100,9 +100,11 @@ delayed_pubkey.
 funding_pubkey.
 4. _remote node's anchor output_: one output paying to the *remote node's*
 funding_pubkey.
-5. _local node's offered HTLCs_: Zero or more pending payments (*HTLCs*), to pay
+5. _shared ephemeral anchor_: one output paying to the *local* or *remote*
+node's funding_pubkey.
```

Comment:
* The ephemeral in the output name isn't used anywhere else, probably just "shared anchor"?
* Description doesn't match the fact that the shared anchor is using P2A

Reply:
This wasn't fully addressed.  It still refers to a keyed anchor instead of P2A.